queries.yaml

```diff
@@ -994,7 +994,7 @@ post_process:
       end_zoom: 11
       items_matching: { kind: locality }
       max_items: 1
-      grid_width: 6
+      grid_width: 18
```

Comment:
To 12?

Reply:
Resolved via https://github.com/tilezen/vector-datasource/pull/2040/commits/e0fe15091eec17cd2bd3cc23dfb853fd2d3d47d1.